### PR TITLE
3.x: Consolidate test logging

### PR DESCRIPTION
**Pull request by benjamintboyle**
Move test logging to consolidated block. Change all 'rootProject' to
'project'.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -120,51 +120,36 @@ jmh {
...
+
         warn.events = ["failed", "skipped"]
-        warn.exceptionFormat="full"
-     }
-} 
+        warn.exceptionFormat = "full"
+    }
 
-check.dependsOn testng 
+    maxParallelForks = Runtime.runtime.availableProcessors().intdiv(2) ?: 1
```
Comment (akarnokd) [inline build.gradle]: This was supposed to run parallel only in non-CI environments.

Comment (benjamintboyle) [reply build.gradle]: I understand that was the intent, but it seems that it has not for quite some time. I wasn't able to find "CI" set anywhere. Should testing on Github be single threaded?

Comment (akarnokd) [reply build.gradle]: https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables
<html>
<body>


Environment variable | Description
-- | --
CI | Always set to true.


</body>
</html>

Comment (benjamintboyle) [reply build.gradle]: Ah, thanks. I wasn't aware of that. I'll submit an update.

**Pull request merged**